common/parser.go

```diff
@@ -65,20 +65,26 @@ func IsValidList(arrayFieldI []interface{}) (listField []interface{}, isValidArr
 
 // ParseEventMessage parses event messages from the kubernetes event log
 func ParseEventMessage(eventType string, resourceName string, resourceKind string, resourceNamespace string, newResourceVersion string, oldResourceVersions ...string) (msg string) {
+	// Support cluster level resources
+	namespacePart := ""
```

Comment:
```suggestion
	inNamespaceMsg := ""
```
This name is clearer (at least to me) for this variable's intention.